go/stellar/stellarsvc/service_test.go

```diff
@@ -843,27 +903,180 @@ func TestRequestPaymentOutsideCurrency(t *testing.T) {
 	require.Equal(t, "$8.20 USD", details.AmountDescription)
 }
 
-// TestImportMakesAccountBundle checks that importing a secret key makes a stellar account
-// bundle (i.e. the new version where there is a bundle per account) and that we
-// can retrieve it from the server.
-func TestImportMakesAccountBundle(t *testing.T) {
+func TestAccountBundleFlows(t *testing.T) {
 	tcs, cleanup := setupNTests(t, 1)
 	defer cleanup()
+	ctx := context.Background()
+	g := tcs[0].G
+	setupWithNewBundle(t, tcs[0])
+
+	bundle, version, _, err := remote.FetchSecretlessBundle(ctx, g)
+	require.NoError(t, err)
+	require.Equal(t, version, stellar1.BundleVersion_V2)
+	accounts := bundle.Accounts
+	secretsMap := bundle.AccountBundles
+	var accountIDs []stellar1.AccountID
```

Comment:
is this used for anything?

Reply:
oops. i missed the expectation that there's 1 account. fixed.